applications/RANSApplication/custom_utilities/rans_variable_utilities.cpp

```diff
@@ -304,6 +304,44 @@ std::tuple<double, double> CalculateTransientVariableConvergence(
     KRATOS_CATCH("");
 }
 
+template <class TContainerType>
```

Comment:
Why do we need to specialize this for conditions? I think that we must stick to the current design criterion, meaning that conditions cannot own a constitutive law. In case a boundary viscous contribution is required, the constitutive law must be accessed through the parent element.

Reply:
Alryt, I'll change it :)

Reply:
done :)